screens/courses/ViewCourseScreen.js

```diff
@@ -23,11 +26,26 @@ class ViewCourseScreen extends React.Component {
       sessions: [],
       course : { },
       isLoading : true,
+      students: { },
     }
   }
 
+  static navigationOptions = ({ navigation }) => {
+    const { params = {} } = navigation.state;
```

Comment:
what the syntax is goin on

Reply:
👀

Reply:
Promise I will fix in next PR when I add redux to Courses :))